G/git_crypt/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder #, Pkg
+
+name = "git_crypt"
+version = v"0.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/AGWA/git-crypt/archive/0.6.0.tar.gz" =>
+    "777c0c7aadbbc758b69aff1339ca61697011ef7b92f1d1ee9518a8ee7702bb78",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd git-crypt-0.6.0/
+export CPPFLAGS=-I${prefix}/include
+make install PREFIX=$prefix
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("git-crypt", :git_crypt)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    # PackageSpec(name="OpenSSL_jll", uuid="458c3c95-2e84-50aa-8efc-19380b2a3a95"), # BB can't handle this ATM
```

Comment:
I think we basically need to eliminate the quotes.  Probably a missing `repr()` somewhere or something.

Reply:
Yea, I have a PR.

Reply:
Although, what I commented on here was another error which show up even if you have the correct syntax

Reply:
Missing import from Pkg?

Reply:
No, iteration of `nothing`, but I will include that in the PR.